privacyidea/lib/policy.py

```diff
@@ -902,6 +985,7 @@ def set_policy(name=None, scope=None, action=None, realm=None, resolver=None,
     :param check_all_resolvers: If all the resolvers of a user should be
         checked with this policy
     :type check_all_resolvers: bool
+    :param conditions: A list of 4-tuples (section, key, comparator, value) of policy conditions
```

Comment:
5-tuples ;-)

Reply:
done in e5951e406cca3de9c09f261c0e16d1920fbaeaa6